portingdb/tables.py

```diff
@@ -139,6 +139,12 @@ def nonblocking(self):
     def list_tracking_bugs(self):
         return [tb.url for cp in self.collection_packages for tb in cp.tracking_bugs]
 
+    @property
+    def last_link_update(self):
+        values = [link.last_update for cp in self.collection_packages
+                for link in cp.links if link.last_update is not None]
+        return values[0] if values else None
```

Comment:
Well, I don't believe there is a rarity of such cases, but I'll conform to your opinion as it is your code base. :)